### ISSUE TITLE
Add pagination, do not load everything into memory

### ISSUE THREAD

**Issue by denisovlev** [Eugene-Mark/bigdata-file-viewer#20]
I was looking at the code and it seems that all the records from the file are loaded into memory. Some files can be quite large. 
Perhaps adding pagination and not loading everything into memory would be better?

**Comment by Eugene-Mark** [comment]
@denisovlev Yes for sure, it's in the to-do list. It's welcomed if you're interested to contribute on it.

**Comment by Eugene-Mark** [comment]
Close it since it's over years, will reopen it if the feature is in the roadmap.
